### DISCUSSION TITLE
Registry pages for packages should not automatically suffix the repository url with .git

### DISCUSSION THREAD

**Discussion by awwong1** [npm/feedback#668] [Registry] [answered]
I notice that when a package is published to https://npmjs.com/ the
repository field is always suffixed with “.git” even if the repository
field in the package.json definition does not contain this suffix.
This occurs using short-form (only url) and long-form (type/url) value.

This breaks on some git providers, including [sourcehut.org](https://sourcehut.org/).

Can we update npmjs such that this behaviour no longer occurs, or is at least configurable?

See also:
- https://lists.sr.ht/~sircmpwn/sr.ht-discuss/%3C5F5D5FEE-3A73-494C-965D-ACEDE8A9B0BA%40gmail.com%3E

**Comment by monishcm** [comment, marked as answer]
Thank you. We will add this to our backlog for investigation. We will contact you if more information is needed.
